crates/autopilot/src/database/competition.rs

```diff
@@ -17,19 +17,19 @@ use {
 
 #[derive(Clone, Debug)]
 pub enum ExecutedFee {
-    /// Fee is calculated by the solver and known upfront (before the settlement
+    /// Fee is determined by user and known upfront (before the settlement
```

Comment:
Is this struct even needed in the co-located world? Can't all fees (fixed order fee as well at the surplus fee) be determined by parsing the settlement in the end? Would this simplify the code paths?